randovania/games/samus_returns/logic_database/Area 6.txt

```diff
@@ -714,8 +714,22 @@ Extra - asset_id: collision_camera_043
   * Extra - actor_name: DoorManicMinerBot
   * Extra - actor_type: doorclosedcharge
   > Before Diggernaut
+      Trivial
+
+> Event - Diggernaut; Heals? False
+  * Layers: default
+  * Event Area 6 - Diggernaut
+  > Pickup (Power Bomb)
+      Trivial
+
+> Before Diggernaut; Heals? False; Default Node
+  * Layers: default
+  * Extra - start_point_actor_name: ST_Checkpoint_009
+  > Door to Electric Escalade
+      After Area 6 - Diggernaut and Lay Power Bomb
+  > Event - Diggernaut
       All of the following:
-          # TODO: move this to event, also have a second look
+          # Damage Diggernauts Bomb Slots
```

Comment:
Damage Diggernaut's Weak Points in the final phase

Reply:
its not just the final pahse tho? its also for the "spider up his legs and put a bomb into its slot"

Reply:
Ah right. `Damage Diggernaut's Weak Points` is sufficient then

Reply:
This wasn't addressed

Reply:
whops didnt read good enough